cobalt/BUILD.gn

```diff
@@ -69,7 +81,40 @@ group("cobalt_browsertests") {
   deps = [ "testing:cobalt_browsertests" ]
 }
 
-if (!is_android) {
+if (is_ios && target_platform == "tvos") {
```

Comment:
Should we add a subplatform gn flag for is_ios_tvos upstream similar to the  BUILDFLAG?